.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
   - id: check-merge-conflict
   - id: check-xml
   - id: check-yaml
+    exclude: ^mkdocs\.yml$
```

Comment:
`format: !!python/name:pymdownx.superfences.fence_code_format` is not valid syntax according this this check-yml check - However, this is the format required for it to work: https://squidfunk.github.io/mkdocs-material/reference/diagrams/#configuration